Integrations/integration-FireEyeHX.yml

```diff
@@ -3043,7 +3134,112 @@ script:
     - contextPath: FireEyeHX.HostSets.url
       description: Host set FireEye url
       type: string
+  description: Get a list of all host sets known to your HX Series appliance
+- name: fireeye-hx-create-indicator
+  arguments:
+    - name: category
+      required: true
+      description: The indicator category
+    - name: create_text
```

Comment:
this argument name is not intuitive at all, if it's the indicator name, name it like that